N/nasm/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "nasm"
```

Comment:
```suggestion
name = "NASM"
```
Note that you must rename the directory accordingly